src/backend/tests/projects.test.ts

```diff
@@ -76,16 +76,47 @@ describe('Projects', () => {
     vi.spyOn(prisma.team, 'findUnique').mockResolvedValue(null);
 
     await expect(
-      async () => await ProjectsService.createProject(batman, 1, 2, 'name', 'summary', ['teamId'])
+      async () =>
+        await ProjectsService.createProject(
+          batman,
+          1,
+          2,
+          'name',
+          'summary',
+          ['teamId'],
+          10,
+          [],
+          [],
+          [],
+          [],
+          [],
+          1, //batman
+          2 //superman
+        )
     ).rejects.toThrow(new NotFoundException('Team', 'teamId'));
   });
 
   test('createProject works', async () => {
     mockGetHighestProjectNumber.mockResolvedValue(0);
     vi.spyOn(prisma.user, 'findUnique').mockResolvedValue(batman);
-    vi.spyOn(prisma.wBS_Element, 'create').mockResolvedValue(prismaWbsElement1);
-
-    const res = await ProjectsService.createProject(batman, 1, 2, 'name', 'summary', []);
+    vi.spyOn(prisma.wBS_Element, 'create').mockResolvedValue({ ...prismaWbsElement1, project: prismaProject2 } as any);
```

Comment:
Do we have to use `any` here?

Reply:
u do if u dont want to declare it before the resolve value